### PR TITLE
refactor: use new table in msp

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
use new table in msp
**1. alarm-strategy** 
**before:**
image

**current:**
image

**2. events in trace-detail**
**before:**
image

**current:**
image

**3. link detail** 
**before:**
image

**current:**
image
 

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/cmp/common/alarm-strategy/index.tsx [hunk 1]
```diff
@@ -153,42 +164,51 @@ export default ({ scopeType, scopeId, commonPayload }: IProps) => {
...
+      deleteStrategy: {
+        title: i18n.t('delete'),
+        onClick: () => {
+          handleDeleteAlarm(record.id);
+        },
+      },
+      enableStrategy: {
+        title: record?.enable ? i18n.t('unable') : i18n.t('enable'),
+        onClick: () => {
+          if (!record?.enable) {
```
Comment (McDaddy) [inline shell/app/modules/cmp/common/alarm-strategy/index.tsx:191]: the code content in if and else is identical. I don't understand the reason

**Comment by daskyrk** [comment]
Badge will have a new style component, need to update.
image

**Comment by sherotree** [comment]
wait for the new Badge Component, will update and pull the request

**Comment by sherotree** [comment]
use the new Badge
image

**Pull request closed without merging**